model_info/imagenet/corruptions/Standard_R50.json

```diff
@@ -7,11 +7,11 @@
   "dataset": "imagenet",
   "venue": "N/A",
   "architecture": "ResNet-50",
-  "eps": null,
-  "clean_acc": "76.52",
+  "eps": 0.03137254901960784,
+  "clean_acc": "76.72",
```

Comment:
I think `"eps": null` was a bit clearer (since there's no threshold involved). Also, do we know why the clean accuracy slightly changed?

Reply:
I agree about `"eps": null`. As per why the clean accuracy slightly changed, could it be because of a different Pillow version (as it happened for #99)?

Reply:
totally agreed about eps=null. fixed it in https://github.com/RobustBench/robustbench/pull/135/commits/754c981b27dc57e6ae37db079e7dec77d743ee1e.

Reply:
> As per why the clean accuracy slightly changed, could it be because of a different Pillow version (as it happened for https://github.com/RobustBench/robustbench/issues/99)?

Yes, sounds very plausible. https://github.com/RobustBench/robustbench/issues/99#issuecomment-1503250499 confirm that.